res/meson.build

```diff
@@ -0,0 +1,6 @@
+meson.add_install_script('meson-install-scripts.sh', get_option('bindir'))
+
+install_data(
+  'org.freedesktop.miracle.conf',
+  install_dir: join_paths('/', get_option('sysconfdir'), 'dbus-1', 'system.d')
```

Comment:
That is correct, end users are expected to set `--sysconfdir=/etc`. Hard-coding directories is not a solution.